src/test/java/io/jenkins/plugins/analysis/core/tokens/IssuesSizeTokenMacroTest.java

```diff
@@ -67,6 +67,15 @@ void shouldExpandTokenOfTwoActions() {
         assertThat(macro.evaluate(run, null, null)).isEqualTo("0");
     }
 
+    @Test
+    void checkTotalNumberOfIssues() {
+        IssuesSizeTokenMacro macro = new IssuesSizeTokenMacro();
+        /**
```

Comment:
Here we would have the setter somewhere like: ` makro.setType(IssueNumberType.NEW)`